### DISCUSSION TITLE
Will the app support Windows 11?

### DISCUSSION THREAD

**Discussion by xNAHPAz** [File-New-Project/EarTrumpet#906] [Q&A] [answered]
Will the app support Windows 11? Because it is currently not working properly.

I'm testing the new Win 11, and consequently downloaded EarTrumpet to use and test. But he is not able to change the volume much less the audio output of the sources.

**Comment by riverar** [comment, marked as answer]
If you download [a dev build](https://eartrumpet.app/), it works. I will try to find some time to get an update out.

**Reply by xNAHPAz** [reply]
Ok! Thanks! I'll try!
